IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -748,6 +748,38 @@
     Z0 = cms.double(0.035748 )
 )
 
+# From Run3 PilotBeams 2021 3.8T data
+# BS parameters extracted from run 346512, fill 7531 (from ExpressPhysics FEVT, Legacy BS):
+# X0         =  0.174282 [cm]
```

Comment:
Out of curiosity: why the parameters written in the comment line do not correspond to the one set in the configuration? What is the meaning of this comment?

Reply:
@perrotta you need to correct for the position of the pixel barycenter in data, as in MC the simulation assumes the detector is centered in the origin. That's the reason of the comments at line 767 and following.

Reply:
> Out of curiosity: why the parameters written in the comment line do not correspond to the one set in the configuration? What is the meaning of this comment?

The parameters in the comment corresponds to the BS positions and widths as computed by the fit. In order to get the proper vertex smearing the BPIX absolute position needs to be subtracted from the BS position.

Reply:
Got it! Thank you!